# Question
with `new Promise((resolve, reject) => {...})`, what do the arguments that you pass into `resolve/reject` do?

# Answer
The argument `resolve` is ran when the promise is resolved, or fulfilled
The argument `reject` is when the promise is rejected, uses the catch method and returns to the user

# Teacher Feedback